src/triage/component/results_schema/nuke_triage.sql

```diff
@@ -0,0 +1,48 @@
+/*
+  Function for wiping all triage tables, schemas and indexes.
+Useful  when ou try to start clean (again) and try to recover some
+previous errors
+ */
+create or replace function nuke_triage()
+    returns text as $result$
+
+    declare
+    result text;
+    query text;
+
+    begin
+
+    execute 'drop schema if exists triage_metadata cascade';
```

Comment:
did `model_metadata` get renamed `triage_metadata` at some point?

Reply:
Oh, yes. Several commits ago... that's why we went to `4.1.0`

Reply:
![](https://media1.giphy.com/media/83QtfwKWdmSEo/200.gif)